packages/react-components/react-persona/MIGRATION.md

```diff
@@ -0,0 +1,41 @@
+# Badge Migration
+
+## Migration from v8
+
+v8 offers a component equivalent to v9's `Persona`. However, the API is slightly different. The main difference is that v9's `Persona` does not handle the functionality of the `presence` and `avatar`. Instead, the `presence` and `avatar` are separate components that can be used in conjunction with `Persona`.
+
+Here's how the API of v8's `Persona` compares to the one from v9's `Persona` component:
+
+- `className` => `className`
+- `coinProps` => Use `avatar`'s or `presence`'s slot props
+- `componentRef` => NOT SUPPORTED - use `ref` instead
+- `hidePersonaDetails` => NOT SUPPORTED
+- `imageShouldFadeIn` => NOT SUPPORTED
+- `isOutOfOffice` => Use `status` in `presence` slot props
```

Comment:
PresenceBadge has an `outOfOffice` prop to use here. And perhaps just write out the syntax for it, in case it's not obvious:
```suggestion
- `isOutOfOffice` => Use the `outOfOffice` prop of the `presence` slot. E.g.: `presence={{ outOfOffice: true }}`
```